tox.ini

```diff
@@ -1,13 +1,36 @@
 [tox]
 envlist =
-    py27,py35,py36,py37,py38,pypy,pypy3,docs
+    py27,py35,py36,py37,py38,pypy,pypy3,coverage,docs
+# Note: if you add new Python versions, please add them to
+# [testenv:coverage] depends as well!
 
 [testenv]
 commands =
-    zope-testrunner --test-path=src
-extras = test
+    coverage run -p -m zope.testrunner --test-path=src []
+extras =
+    test
+deps =
+    coverage
+setenv =
+     ZOPE_INTERFACE_STRICT_IRO=1
```

Comment:
The inndentation here isn't consistent with the other options.  Looks like 5 spaces to me?

Reply:
Thanks. No doubt an artifact of copy-n-paste and comment/uncomment a few times.

(I don't have emacs specially configured to do what we usually do in `tox.ini` indentation-wise, so it takes manual counting. Emacs wants to line things up with the `=` by default.)